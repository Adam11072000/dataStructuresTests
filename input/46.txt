init 38
addImage 221
addImage 221
resetLabelScore 237 29 125
addImage 242
resetLabelScore 65 37 45
addImage 242
quit