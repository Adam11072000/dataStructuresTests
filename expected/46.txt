init done.
resetLabelScore: FAILURE
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
addImage: SUCCESS
deleteImage: SUCCESS
getHighestScoredLabel: FAILURE
getHighestScoredLabel: INVALID_INPUT
addImage: INVALID_INPUT
quit done.
